### DISCUSSION TITLE
cannot find en.json

### DISCUSSION THREAD

**Discussion by salubay** [mozilla/hubs#5038] [General Questions & Discussion] [answered]
I'd like to translate to Turkish. I couldn't find en.json file.

**Comment by matthewbcool** [comment, marked as answer]
Thanks for your interest! To generate the en.json you must run the command: `npm run intl-extract-en-locale`

Instructions here: https://github.com/mozilla/hubs/blob/master/src/assets/locales/README.md

Video: https://youtu.be/kmjZSVjtxLA
